src/scraper/tests/el_pitazo_test.py

```diff
@@ -0,0 +1,36 @@
+# TODO move tests out of this module when approved.
+
+from scraper.tests.utils import fake_response_from_file
+from scraper.scrapers.el_pitazo_scraper import ElPitazoScraper
+from scraper.settings import ROOT_DIR
+import os
+
+
+def test_parse_ok():
+    """
+        Check that ElPitazoScraper parses a valid page as expected
+    """
+    url = "tests/html_bodies/el_pitazo_fallas_electricas_carne.html"
```

Comment:
let's try to put anything that is a resource inside a resources folder. That is more pythonic. Or perform a quick search for recommended practices for resources in pytest. I think moving the html_bodies folder inside a resources folder should be enough.